libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SalesforceNetworkPlugin.java

```diff
@@ -120,8 +120,12 @@ protected void sendRequest(JSONArray args, final CallbackContext callbackContext
             }
             restClient.sendAsync(request, new RestClient.AsyncRequestCallback() {
 
+                // TODO: Remove onSuccess for 9.0 Release.
                 @Override
-                public void onSuccess(RestRequest request, RestResponse response) {
+                public void onSuccess(RestRequest request, RestResponse response) {}
```

Comment:
There's no point "deprecating" this interface method (the customer still has to implement the new callback that we have introduced which makes it a breaking change). I'd make an exception and simply rename it to `onResponse`.